### ISSUE TITLE
Using xG instead of goals

### ISSUE THREAD

**Issue by goktugerce** [Torvaney/mezzala#4]
Since Dixon-Coles model uses Poisson distribution I guess it is not possible (or easy) to use xG values instead of actual goals to train the model. I tried multiplying xG values by 100 and then rounding, and it gave team strengths just fine (I guess) but simulating the games did not work correctly (the outcomes become 0).

Just wanted to learn if it is possible to somehow integrate xG into this. I came across your [Dixon Coles and xG: together at last](https://www.statsandsnakeoil.com/2018/06/22/dixon-coles-and-xg-together-at-last/) blog post and it suggests we can incorporate xG simulations into this by "tricking" the model. Personally I could not come up with a way to do it in Python (mainly because I did not understand what to pass as `weights` parameter).

Some help is appreciated if you are still maintaining this.

**Comment by goktugerce** [comment]
After giving it a better read, I understood that it's not only about the parameters but also about the data points we pass to the model. We will not have 380 games for an EPL season but have -let's say- 36 rows for each game with assigned probabilities.

I will give this a shot tomorrow, hopefully I can also integrate time-based weighting into this. Will keep this space updated.

**Comment by Torvaney** [comment]
Yes, exactly; you can see an example project using this method with this package here: https://github.com/Torvaney/wingback/blob/master/wingback/team_strength.py
